README.rst

```diff
@@ -1,12 +1,12 @@
-`luma.core <https://github.com/rm-hull/luma.core>`__ **|** 
-`luma.docs <https://github.com/rm-hull/luma.docs>`__ **|** 
-`luma.emulator <https://github.com/rm-hull/luma.emulator>`__ **|** 
-`luma.examples <https://github.com/rm-hull/luma.examples>`__ **|** 
-luma.lcd **|** 
-`luma.led_matrix <https://github.com/rm-hull/luma.led_matrix>`__ **|** 
-`luma.oled <https://github.com/rm-hull/luma.oled>`__ 
-
-Luma.LCD
+`luma.core <https://github.com/rm-hull/luma.core>`__ **|**
+`luma.docs <https://github.com/rm-hull/luma.docs>`__ **|**
+`luma.emulator <https://github.com/rm-hull/luma.emulator>`__ **|**
+`luma.examples <https://github.com/rm-hull/luma.examples>`__ **|**
+luma.lcd **|**
+`luma.led_matrix <https://github.com/rm-hull/luma.led_matrix>`__ **|**
+`luma.oled <https://github.com/rm-hull/luma.oled>`__
+
+luma.lcd
 ========
 **PCD8544, ST7735, ST7567, HT1621, UC1701X, ILI9341 Display Drivers**
```

Comment:
missing HD44780

Reply:
Added HD44780.  I also added a picture of the HD44780 module.  I notice that the rest of the display pictures were powered off.  Let me know if you want me to replace this image with one that is consistent with the others.